zellij-server/src/screen.rs

```diff
@@ -702,21 +702,29 @@ impl Screen {
     }
 
     pub fn add_client(&mut self, client_id: ClientId) {
-        let mut tab_index = 0;
```

Comment:
```suggestion
let mut tab_index = *self.tabs.keys().next().expect("no tabs found");
```

I think this is maybe easier than changing the logic of the ifs below?